client/me/help/help-contact/index.jsx

```diff
@@ -481,7 +481,7 @@ const HelpContact = React.createClass( {
 			},
 			showHappychatVariation && {
 				onSubmit: this.startHappychat,
-				buttonLabel: this.translate( 'Chat with us' )
+				buttonLabel: ( config( 'env' ) === 'development' ) ? 'Happychat with us' : this.translate( 'Chat with us' ) 
```

Comment:
Happychat is Happychat in all languages, like Taxi.
Also it only shows in dev or staging environment which is most likely Automatticians